tests/main/netplan-apply/task.yaml

```diff
@@ -0,0 +1,67 @@
+summary: Ensure that netplan apply works with network-setup-control
+
+details: |
+    Netplan apply is used to apply network configuration to the system
+
+# run on all classic ubuntu LTS and current dev systems 16.04+
+systems: 
+    - ubuntu-16*
+    - ubuntu-18*
+    - ubuntu-19*
+    - ubuntu-20*
+
+prepare: |
+    #shellcheck source=tests/lib/snaps.sh
+    . "$TESTSLIB"/snaps.sh
+    snap install test-snapd-netplan-apply --edge
+
+    # ensure that the service file doesn't exist before running
+    rm -f /usr/share/dbus-1/system-services/io.netplan.Netplan.service
```

Comment:
Perhasps you could rename/backup these files and restore them during the restore, so you make sure the configuration will be the same than before the test is executed.

Reply:
Done